pyproject.toml

```diff
@@ -13,6 +13,7 @@ skip_gitignore = true
 extend_skip_glob = ['venv/*', 'docs/*']
 
 [tool.mypy]
+python_version = "3.7"
```

Comment:
This should help a bit with getting consistent errors across dev environments.
If you are using a python `3.10` feature that doesn't exist in `3.7` mypy should warn you.